zipkin-cassandra/src/main/scala/com/twitter/zipkin/cassandra/CassieSpanStoreFactory.scala

```diff
@@ -15,102 +15,22 @@
  */
 package com.twitter.zipkin.cassandra
 
-import com.twitter.app.{App, Flaggable}
-import com.twitter.cassie._
-import com.twitter.cassie.connection.{CCluster, RetryPolicy}
-import com.twitter.concurrent.Spool
-import com.twitter.concurrent.Spool.*::
+import com.datastax.driver.core.Cluster;
+import com.twitter.app.App
 import com.twitter.conversions.time._
 import com.twitter.finagle.{Addr, Name, Resolver}
-import com.twitter.finagle.builder.Cluster
 import com.twitter.finagle.stats.{DefaultStatsReceiver, StatsReceiver}
-import com.twitter.finagle.tracing.DefaultTracer
-import com.twitter.finagle.util.InetSocketAddressUtil
-import com.twitter.util.{Await, Future, Promise, Return, Var}
 import com.twitter.zipkin.thriftscala.{Span => ThriftSpan}
 import com.twitter.zipkin.storage.cassandra._
-import java.net.SocketAddress
-import scala.collection.mutable.HashSet
 
 trait CassieSpanStoreFactory { self: App =>
```

Comment:
+1
 (was intentionally leaving it until later in the PR lifecycle so the diff on the file remains clear)